src/components/PostItem/PostItem.tsx

```diff
@@ -0,0 +1,53 @@
+import { FC } from 'react';
+import cn from 'classnames';
+import { Post } from '../../types/Post';
+
+interface Props {
+  post: Post,
+  selectedPostId: number,
+  onPostSelect: (post: number) => void
```

Comment:
That name is better reflect the value in it
```suggestion
  onPostSelect: (postId: number) => void
```